app/assets/v2/js/pages/new_bounty.js

```diff
@@ -105,39 +104,6 @@ Vue.mixin({
         console.log(err);
       });
     },
-    injectProvider: function(token) {
-      let vm = this;
-      const chainId = vm.chainId;
-
-      if (!token || !chainId) {
-        return;
-      }
-
-      switch (chainId) {
-        case '59':
-        case '58': {
-          let polkadot_endpoint;
-
-          if (token == 'KSM') {
-            polkadot_endpoint = KUSAMA_ENDPOINT;
-          } else if (token == 'DOT') {
-            polkadot_endpoint = POLKADOT_ENDPOINT;
-          }
-
-          polkadot_utils.connect(polkadot_endpoint).then(res =>{
-            console.log(res);
-            polkadot_extension_dapp.web3Enable('gitcoin');
-          }).catch(err => {
-            console.log(err);
-          });
-          break;
-        }
-
-
-        default:
-          break;
-      }
-    },
```

Comment:
Are we injecting polkadotjs somewhere else? Why was this removed?

Reply:
This not needed on bounty creation !
It's just additional overhead and adds more friction 

This was originally added under the assumption that they would pay fees while bounty was been created
Now that we've decided we would not charge fee -> it makes sense to remove this

Reply:
Thats awesome!! Thank you!